test/tests/selftest/pbs_pbstestsuite.py

```diff
@@ -268,3 +268,18 @@ def test_revert_pbsconf_fewer_vars(self):
         # Confirm that the variable was set again
         pbs_conf_val = self.du.parse_pbs_config(self.server.hostname)
         self.assertIn("PBS_CORE_LIMIT", pbs_conf_val)
+
+    def test_revert_moms_default_conf(self):
+        """
+        Test if PBSTestSuite.revert_moms() reverts the mom configuration
+        setting to defaults
+        """
+        a = {'$prologalarm': 280}
+        self.mom.add_config(a)
+
+        c1 = {'$prologalarm': '280',
+              '$clienthost': self.server.hostname}
+        self.assertEqual(self.mom.parse_config(), c1)
+        self.mom.revert_to_defaults()
+        c2 = {'$clienthost': self.server.hostname}
+        self.assertEqual(self.mom.parse_config(), c2)
```

Comment:
Shouldn't you actually see if "prologalarm" is not still set?

Reply:
I don't think so. The dictionary returned by parse_config() is the latest copy of config. Besides, "I think", there is currently no mechanism to 'expect' the mom configuration parameter values in PTL